docs/sources/installation/restart-grafana.md

```diff
@@ -0,0 +1,106 @@
++++
+title = "Restart Grafana"
+description = "Installation guide for Grafana"
+keywords = ["grafana", "restart", "documentation"]
+weight = 750
++++
+
+# Restart Grafana
+
+Users often need to restart the Grafana service after they have made configuration changes. This topic provides detailed instructions on how to restart the service on supported operating systems.
+
+- [Windows](#windows)
+- [MacOS](#macos)
+- [Linux](linux)
+- [Docker](#docker)
+
+## Windows
+
+To restart Grafana:
+
+1. Open the Services app.
+2. Right-click on the **Grafana** service.
```

Comment:
```suggestion
1. Right-click on the **Grafana** service.
```

Reply:
In my Windows 10 VM appears as `grafana-server.exe` so I think that it can confuse the users.

Reply:
@oddlittlebird and @papagian, what should be the instruction then? LMK so I can update the doc. TY.

Reply:
In my non-VM Windows, it is just "Grafana." I think we can have a little faith in our users here that they can figure out that grafana-server.exe is the Grafana service.